code/modules/mob/living/simple_animal/hostile/megafauna/swarmer.dm

```diff
@@ -167,7 +167,7 @@ GLOBAL_LIST_INIT(AISwarmerCapsByType, list(/mob/living/simple_animal/hostile/swa
 	attack_all_objects = TRUE //attempt to nibble everything
 	lose_patience_timeout = 150
 	var/static/list/sharedWanted = typecacheof(list(/turf/closed/mineral, /turf/closed/wall)) //eat rocks and walls
-	var/static/list/sharedIgnore = list()
+	var/static/list/sharedIgnore = list(/obj/machinery/mineral/ore_redemption, /obj/machinery/rnd/production/protolathe, /obj/machinery/rnd/production/circuit_imprinter/department, /obj/machinery/ore_silo)
```

Comment:
you could remove the swarmer_act() from these machines since swarmers check if the thing they are attacking has that proc, and if it doesn't have it add it to this list. 
This is just snowflake

Reply:
Yup yup yup, someone pointed it out to me just now. I'm just dumb